tools/devtool

```diff
@@ -72,7 +72,7 @@
 DEVCTR_IMAGE_NO_TAG="public.ecr.aws/firecracker/fcuvm"
 
 # Development container tag
-DEVCTR_IMAGE_TAG="v33"
+DEVCTR_IMAGE_TAG="kernel_support"
```

Comment:
Is this intentional ?

Reply:
yes, I used a development image for this PR.

In my next push, I am replacing it with v34 (the next one after the current one).